.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @ngam @farhantejani @ghego @h-vetinari @hajapy @hmaarrfk @jschueller @njzjz @waitingkuo @wolfv @xhochy
\ No newline at end of file
+* @farhantejani @ghego @h-vetinari @hajapy @hmaarrfk @jschueller @ngam @njzjz @waitingkuo @wolfv @xhochy
```

Comment:
this seems problematic. did you rebase on recent main?

Reply:
Yes, but that change is from the bot itself, see 1688b3bfdd5d349eeadf7a9ed12b014642d12201.

AFAICT, the issue is that the `add user @xyz` stuff tries to skip CI and so just adds the user in front of the `CODEOWNERS`, whereas conda smithy orders the owners alphabetically